runelite-client/src/main/java/net/runelite/client/plugins/woodcutting/WoodcuttingConfig.java

```diff
@@ -87,4 +87,16 @@ default boolean showRespawnTimers()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+			keyName = "RespawnTimerSize",
+			name = "Size of Respawn timers",
+			description = "Change the size of the Respawn timers",
+			position = 10
```

Comment:
These lines are indented one level too far.
```suggestion
		keyName = "RespawnTimerSize",
		name = "Size of Respawn timers",
		description = "Change the size of the Respawn timers",
		position = 10
```

Reply:
Also the position for this added config should be 6, not 10.